### ISSUE TITLE
MapperParsingException object mapping for isPartOf.hasSuperordinate

### ISSUE THREAD

**Issue by dr0i** [hbz/lobid-resources#503]
Around 100 docs are dropped, e.g.:

> [2017-07-03 00:36:10,101][DEBUG][action.bulk              ] [DE-605_gaia] [resources-201707
02-0100][4] failed to execute bulk item (index) index {[resources-20170702-0100][resource][
TT000130499], source[{"@context":"http://lobid.org/resources/context.jsonld","contribution"
:[{"agent":{"altLabel":"Junghann, Elisabeth Fischer-","id":"http://d-nb.info/gnd/105592021"
,"label":"Fischer-Junghann, Elisabeth","type":["Person"]},"role":{"id":"http://id.loc.gov/v
ocabulary/relators/cre","label":"Autor/in"},"type":["Contribution"]}],"contributorOrder":["
http://d-nb.info/gnd/105592021"],"describedBy":{"dateCreated":"19980424","id":"http://lobid
.org/resources/TT000130499","label":"lobid Ressource"},"extent":"86 S.","hbzId":"TT00013049
9","id":"http://lobid.org/resources/TT000130499#!","isPartOf":[{"hasSuperordinate":"http://
lobid.org/resources/TT000130499#!","numbering":"9","type":["IsPartOfRelation"]}],"label":["
Musikpädagogische Bibliothek"],"medium":[{"id":"http://rdaregistry.info/termList/RDAproduct
ionMethod/1010","label":"Print"}],"otherTitleInformation":["der Gesang am Scheidewege"],"pu
blication":[{"location":"Heidelberg","publishedBy":"Quelle & Meyer","startDate":"1964","typ
e":["PublicationEvent"]}],"responsibilityStatement":["Elisabeth Fischer-Junghann"],"sameAs"
:[{"id":"http://hub.culturegraph.org/resource/HBZ-TT000130499","label":"Culturegraph Ressou
rce"}],"title":"Gesangsbidungslehre","type":["BibliographicResource","Book"]}]}
MapperParsingException[object mapping for [isPartOf.hasSuperordinate] tried to parse field [hasSuperordinate] as object, but found a concrete value]
--

**Comment by dr0i** [comment]
The cause is the missing `id` field in `hasSuperordinate.id`.

**Comment by ChristophEwertowski** [comment]
Will be worked on after the deployment of #502.

**Comment by ChristophEwertowski** [comment]
There resources are labeled as parts of themselves: http://lobid.org/hbz01/TT000136775 (MAB 001 / MAB 453). Also resources exist which only have an id and no label: http://lobid.org/hbz01/TT000134440

**Comment by ChristophEwertowski** [comment]
Because of the high number of above 100 resources I'll make a list and give them our catalogers. I suppose that most of them are, like the examples above, part of a series which we already have in the MAB data.

**Comment by ChristophEwertowski** [comment]
I grepped the resources which have their own id as a superordinateId and put them into a file on the server. ~440 resources have this problem. Of them only 8 have HT-numbers, the rest start with TT and thus are imports. I talked to Mrs. Büning about it but she told me that she has too few resources to tackle this problem. If Mrs. Block returns I will talk to the data import provider so that in the future we don't have this problem anymore. So it has to wait until I get the introduction to Aleph. (See #258)
